ant/windows/windows-packager.nsi.in

```diff
@@ -53,6 +53,16 @@ Section
   DetailPrint "Cleaning up resources from previous versions..."
   RMDir /r "$INSTDIR\demo\js\3rdparty"
   Delete "$INSTDIR\demo\js\qz-websocket.js"
+  Delete "$SMSTARTUP\${project.name}.lnk"
+
+  ; Remove 2.1 startup entries
+  ReadEnvStr $R0 "APPDATA"
+  IfFileExists "$R0\${vendor.name}\.autostart" 0 +2
+  Delete "$R0\${vendor.name}\.autostart"
+
+  ReadEnvStr $R0 "PROGRAMDATA"
+  IfFileExists "$R0\${vendor.name}\.autostart" 0 +2
+  Delete "$R0\${vendor.name}\.autostart"
```

Comment:
@klabarge I think you've pointed out a discrepancy with the `.autostart` preference files in 2.1 being cleaned up on one platform (Windows) but not on the others (Linux, Mac).  For consistency you can drop this logic, or keep it, it's up to you.  It may make sense to file a bug report against 2.1 for this inconsistency so that the installers behave the same across all platforms.

Reply:
>  It may make sense to file a bug report against 2.1 for this inconsistency so that the installers behave the same across all platforms.

@tresf This inconsistency is purely on my end. All 3 platforms create similar files.

Initially I was just going to add the cleanup of the `.autostart` files on macOS and Linux as well, but  I'm just going to remove the logic on Windows. They don't harm anything being present in 2.0, and it's probably a bad idea to blow these away (in 2.1) on install anyway. Each time the software gets updated, the autostart exceptions on a per-user bases would be deleted.